src/cli.ts

```diff
@@ -16,19 +28,32 @@ module.exports = function cli(commands) {
 
     .command(commands)
 
-    .option('verboseErrors', builders.verboseErrors)
-    .option('suppressWarnings', builders.suppressWarnings)
-
+    .option('verboseErrors', {
+      alias: ['v', 'verbose'],
+      type: 'boolean',
+      describe: 'Display original errors',
+      required: false,
+      default: false,
+    })
+    .option('suppressWarnings', {
+      alias: 'w',
+      type: 'boolean',
+      describe: 'Suppress warnings',
+      required: false,
+      default: false,
+    })
+    
     .strict()
 
-    .middleware((argv) => {
+    .middleware((argv: any) => {
       assignGlobals(argv);
       sanitizeInputs(argv);
     })
 
-    .fail((msg, err, _yargs) => {
+    .fail((msg: string, err: Error, _yargs: any) => {
```

Comment:
We should use `unknown` instead of `any` here, just like wherever it's possible to do so.